### PR TITLE
[EASI-2609] Addressing 508 findings

### PR DESCRIPTION
**Pull request by garyjzhao**
# EASI-2609

## Changes and Description

- Add various aria tags to different components mentioned in the Findings document
- Renamed `HelpTag` to `HelpAndKnowledgeCategoryTag`, so it is way more specific/readable
- add `ariaLabel` prop in `MultiSelect` component as a required prop

## How to test this change

- verify screen reader is reading out the accessibiltiy things mentioned in the doc

## PR Author Review Checklist

- [ ] Met the ticket's acceptance criteria, or will meet them in a subsequent PR.
- [ ] Added or updated tests for backend resolvers or other functions as needed.
- [ ] Added or updated client tests for new components, parent components, functions, or e2e tests as necessary.
- [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.

### CODE REVIEW

**Review by patrickseguraoddball — CHANGES_REQUESTED** [review]
Thanks for addressing these Gary.

At the time of these findings, IT Solutions help center wasn't implemented, and could use some better aria-labeling.  

`Learn more` links
Screenshot 2023-04-24 at 11 01 10 AM

`Category tags`
Screenshot 2023-04-24 at 11 01 44 AM

`About this solution links`
Screenshot 2023-04-24 at 11 02 05 AM

This no results item has no tabIndex/ not tabbable
Screenshot 2023-04-24 at 11 04 54 AM

I think the star buttons should have a unique label, maybe the model plan name appended to the end
Screenshot 2023-04-24 at 11 21 17 AM

I think the results of seach need an `aria-live` attribute for screen readers to announce results
Screenshot 2023-04-24 at 11 22 49 AM

File: src/views/ModelPlan/TaskList/OpsEvalAndLearning/Evaluation/__snapshots__/index.test.tsx.snap [hunk 1]
```diff
@@ -492,6 +494,7 @@ exports[`Model Plan Ops Eval and Learning matches snapshot 1`] = `
         class="usa-label maxw-none"
         data-testid="label"
         for="ops-eval-and-learning-data-to-send"
+        id="ops-eval-and-learning-data-to-send"
```
Comment (ClayBenson94) [inline src/views/ModelPlan/TaskList/OpsEvalAndLearning/Evaluation/__snapshots__/index.test.tsx.snap:497]: I think adding this ID makes us have 2 IDs that are the same on this page: https://github.com/CMSgov/mint-app/blob/6ec9517315c4e0963e7e3fbfa45acb2b6c1b95c8/src/views/ModelPlan/TaskList/OpsEvalAndLearning/Evaluation/index.tsx#L517

I think that's what's breaking the E2E test here https://github.com/CMSgov/mint-app/blob/6ec9517315c4e0963e7e3fbfa45acb2b6c1b95c8/cypress/e2e/opsEvalAndLearning.spec.js#L285-L287


image

**Review by ClayBenson94 — CHANGES_REQUESTED** [review]

**Review by patrickseguraoddball — CHANGES_REQUESTED** [review]
Thanks Gary, just two outstanding things

There should be a tabindex on the text that reads `No results` here, so it is tabbable, but not selectable.  Might need to look into the library docs for adjusting this functionality

Screenshot 2023-05-01 at 3 02 43 PM

There should be an aria-live label on this warning if no results are found from the query
Screenshot 2023-05-01 at 3 04 38 PM

**Review by patrickseguraoddball — APPROVED** [review]
LG

**Pull request merged**